Dockerfile

```diff
@@ -253,4 +287,9 @@ COPY --from=nnpy /output/usr/local /usr/local/
 COPY --from=thrift /output/usr/local /usr/local/
 COPY --from=protobuf /output/usr/local /usr/local/
 COPY --from=grpc /output/usr/local /usr/local/
+# `pip install --user` will place things in `site-packages`, but Ubuntu expects
+# `dist-packages` by default, so we need to set configure `site-packages` as an
+# additional "site-specific directory".
```

Comment:
Seriously? I thought it is looking at both by default ...

Reply:
I wish it were not so. Ubuntu somehow managed to make an already confusing system even worse.